plugins/broker-mqtt/src/tests/__snapshots__/createControllerFromTopics.spec.ts.snap

```diff
@@ -0,0 +1,51 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Testing the creation of topic subscription in controller should create a controller with topics if resources are found 1`] = `
+[
+  {
+    "code": "import { Controller } from "@nestjs/common";
+import { Ctx, EventPattern, MqttContext, Payload } from "@nestjs/microservices";
+
+@Controller("mqtt-controller")
+export class MqttController {
+  @EventPattern(test-topic-name)
```

Comment:
The parameter must be a string of the topic name or the Enum value for the topic itself
```suggestion
  @EventPattern("test-topic-name")
```